test/RelayHub.test.ts

```diff
@@ -1257,8 +1259,9 @@ contract('RelayHub', function ([_, relayOwner, relayManager, relayWorker, incorr
       env = await getTestingEnvironment()
       chainId = env.chainId
 
-      penalizer = await Penalizer.new()
-      relayHubInstance = await deployHub(penalizer.address)
+      relayHubInstance = await deployHub()
+      penalizer = await Penalizer.new(relayHubInstance.address)
+      await relayHubInstance.setPenalizer(penalizer.address)
```

Comment:
If possible, I'd create a utility function to deploy both `RelayHub` and `Penalizer`